### ISSUE TITLE
[Feature Request] detect dataview styled fields

### ISSUE THREAD

**Issue by AtlcatlQ** [mdelobelle/metadatamenu#28]
I'd like dataview styled field and other not isolated inline fields can be modified using this plugin. The current behavior is:

``` Markdown
field:: Works

- field2:: doesn't work

[field3:: Doesn't work either]

*field4*:: But this works
```

I would like to all works.

**Comment by mdelobelle** [comment]
closed with #47

**Comment by AtlcatlQ** [comment]
Thanks, That's great!!
